UI/Web/src/theme/components/_sidenav.scss

```diff
@@ -227,7 +227,7 @@
           text-align: unset;
           margin-left: 0.75rem;
           font-size: 0.9rem;
-          color: #999999;
+          color: var(--side-nav-text-color, #999999);
```

Comment:
@majora2007 I just noticed that this is the only place in this file where you weren't using `--side-nav-text-color`. This color is different from the one in `dark.scss`. Not sure what you prefer to do here.

https://github.com/Kareadita/Kavita/blob/d4e3a2de3ea3e681052dec8d9a9fe405a61c970a/UI/Web/src/theme/themes/dark.scss#L247

Reply:
Hmm, I also see `color: #d5d5d5;` for the preference side nav (which I'm sure you'll want to scope with a css variable). 

As for the main nav, let's stick with `#999`.

Reply:
> Hmm, I also see `color: #d5d5d5;` for the preference side nav

This one is actually being overridden by this part in `_sidenav.scss`, so the preference nav headers are white.

https://github.com/Kareadita/Kavita/blob/a3c8c9be333e947b3931813f01cac4326b7f1dcc/UI/Web/src/theme/components/_sidenav.scss#L205-L206

I can either remove the `color` property from the code referenced above entirely, or I can just set it to the same `--side-nav-header-text-color` variable

Reply:
For the preference side nav, let's just introduce a new css variable `--pref-side-nav-header-text-color`. I want to keep them as 2 different colors.